wrims/build.gradle

```diff
@@ -0,0 +1,56 @@
+plugins{
+    id 'library.deps-conventions'
+    id 'library.java-conventions'
+
+    // The following plugin is used to build the OSGI bundle
+    id 'biz.aQute.bnd.builder' version '6.4.0'
+}
+
+repositories {
+    mavenLocal()
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+sourceSets {
+    src {
+        java {
+            srcDirs "src/main/java/"
+        }
+    }
+}
+
+dependencies {
+    implementation project(':wrims-core')
+
+    implementation libs.commons.io
+    implementation libs.jfreechart
+    implementation libs.hec.monolith
+    implementation libs.hec.nucleus.metadata
+
+    implementation fileTree(dir: 'libs/eclipse', include: ['*.jar'])
```

Comment:
Seems like these jars should maybe be in a separate configuration, then `implementation.extendsFrom(eclipse)`

Down on line 49, I believe doing it this way would allow you to do `from(configurations.runtimeClasspath - configurations.eclipse)` which is a bit more clear than excluding jars by a filename

Reply:
I followed your suggestion and added the eclipse jars to an eclipseLibs configuration for clarity.